x/group/module/module.go

```diff
@@ -63,6 +72,29 @@ func (a Module) GetQueryCmd() *cobra.Command {
 	return nil
 }
 
+func (a Module) GenerateGenesisState(input *module.SimulationState) {
+}
+
+func (a Module) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
+	return nil
+}
+
+func (a Module) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
+	return nil
+}
+
+func (a Module) RegisterStoreDecoder(registry sdk.StoreDecoderRegistry) {
+}
+
+func (a Module) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
+	protoCdc := codec.NewProtoCodec(a.Registry)
+	// TODO: WeightedOperations requires group module query-server
+	return simulation.WeightedOperations(
+		simState.AppParams, simState.Cdc,
+		a.AccountKeeper, a.BankKeeper, protoCdc,
```

Comment:
Is there any way to access group module msgServer here @aaronc?

Reply:
We can use the MsgClient with text fixtures. Do you need private state here?

Reply:
Each simulation operation can depend on the previous one. In the case of the group module, to create a group account 
I don't think . We need a way to pass `group.QueryClient` [here](https://github.com/regen-network/regen-ledger/pull/251/files#diff-e972a3eebdba55ab7bc12235f56652e406f5bc2229c8b4bbe573a2479e9416cdR92).

Reply:
The whole simulation state is initialized in `app/sim_test.go`. Mostly we are querying the group module state, we need a way to pass `group.QueryClient` from [here](https://github.com/regen-network/regen-ledger/pull/251/files#diff-e972a3eebdba55ab7bc12235f56652e406f5bc2229c8b4bbe573a2479e9416cdR92).
Some context:  Each simulation operation can depend on the previous one. In the case of the group module, to create a group account we need [group-info](https://github.com/regen-network/regen-ledger/pull/251/files#diff-c7b2e4ef3c1b56b3e1be6efbd7d9ed108b8f3457089b7344692be5fe82733608R145) (which is created in the previous operation [SimulateMsgCreateGroup](https://github.com/regen-network/regen-ledger/pull/251/files#diff-c7b2e4ef3c1b56b3e1be6efbd7d9ed108b8f3457089b7344692be5fe82733608R79)).